src/app/meeting/write/page.tsx

```
@@ -0,0 +1,199 @@
+  // 운영진만 모임 등록 가능
+  if (!isStaff) {
+    return (
+      <RequireMember>
+        <main className="container-x-lg flex min-h-[60vh] flex-col items-center justify-center text-center">
+          <Mascot emotion="sad" size="md" />
+          <h1 className="mt-6 text-xl font-bold text-gray-900">운영진만 등록할 수 있어요</h1>
+          <p className="mt-2 text-sm text-gray-500">모임 일정은 운영진이 등록합니다.</p>
+          <button
+            onClick={() => router.push("/meeting")}
+            className="mt-6 rounded-full border border-gray-200 px-6 py-2.5 text-sm text-gray-600 transition-colors hover:bg-gray-50"
+          >
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)